XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -222,6 +236,14 @@ private void EnabledRow(Host host, UpdateType type, int index)
                     return;
                 }
 
+                
+                if (poolOfOne != null && PoolForbidsAutomaticUpdates(poolOfOne))

```

Comment:
I'm happy to rethink this too, but I didn't want to change any of the current behaviour by overriding error messages.